fleet-management-spring-mvc/pom.xml

```diff
@@ -19,11 +19,6 @@
             <artifactId>fleet-management-sample-data</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
-        <dependency>
-            <groupId>cz.fi.muni.pa165</groupId>
-            <artifactId>fleet-management-rest</artifactId>
-            <version>${project.parent.version}</version>
```

Comment:
I think that rest and spring mvc should be two independent modules not mutually dependent running at the same time. Like this it was implemented in the sample project as well as in the last year projects... but if you come up with another design that will work, it will be ok :-)